bin/docs.test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -rf sandbox/loopback.io/
```

Comment:
Should we consider adding checks for status codes returned by those commands or would Travis handle it if any of them fail (I guess they are interdependent so maybe this is not needed)?

Reply:
I think @b-admike's point was that the shell script only reports the last command's exit code to travis.